src/design-system/components/input-field/InputField.tsx

```diff
@@ -5,23 +5,26 @@ import ClearIcon from '@mui/icons-material/Clear'
 import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline'
 import { cn } from '~/utils/cn'
 
-import { InputBaseProps } from '@mui/material/InputBase'
+import InputBase, { InputBaseProps } from '@mui/material/InputBase'
 
 import Box from '@mui/material/Box'
 import { SxProps } from '@mui/material'
 
-export interface InputFieldProps extends InputBaseProps {
-  variant: InputFieldVariantEnum
+type BaseInputFieldProps = {
+  variant?: InputFieldVariantEnum
```

Comment:
If you don`t change enum dist, please make pull for this branch. I replaced your enum to constants.ts